  INFO root [2018-08-08 18:38:14,716] - 
  INFO root [2018-08-08 18:38:14,717] - -----------------------------------------------------------------------------------------


  INFO root [2018-08-08 18:38:14,719] - Resmgr::initialize: KBE_ROOT=D:/KBE/kbengine-2.2.0/
  INFO root [2018-08-08 18:38:14,719] - Resmgr::initialize: KBE_RES_PATH=D:/KBE/kbengine-2.2.0/kbe/res/;D:/KBE/kbengine-2.2.0/assets/;D:/KBE/kbengine-2.2.0/assets/scripts/;D:/KBE/kbengine-2.2.0/assets/res/
  INFO root [2018-08-08 18:38:14,720] - Resmgr::initialize: KBE_BIN_PATH=D:/KBE/kbengine-2.2.0/kbe/bin/server/
  INFO root [2018-08-08 18:38:14,776] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.108 ( 192.168.0.108:62414 )
  INFO root [2018-08-08 18:38:14,777] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.108:62414, SOMAXCONN=5.
  INFO root [2018-08-08 18:38:14,777] - server-configs:
  INFO root [2018-08-08 18:38:14,778] - 	internalTcpAddr : 192.168.0.108:62414
  INFO root [2018-08-08 18:38:14,778] - 	componentID : 4000
  INFO root [2018-08-08 18:38:16,298] - ---- dbmgr Version: 2.2.0. ScriptVersion: 0.1.0. Protocol: C85D8893D1079B3F8EA4D6F8485316E8. Config: Debug X86. Built: 18:34:10 Aug  8 2018. AppID: 4000. UID: 9028. PID: 13348 ----
  INFO root [2018-08-08 18:38:16,314] - Server : Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz with 4194303kB RAM
  INFO root [2018-08-08 18:38:16,318] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-08-08 18:38:16,319] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-08-08 18:38:16,557] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-08-08 18:38:16,558] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-08-08 18:38:16,559] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-08-08 18:38:16,567] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-08-08 18:38:16,568] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-08-08 18:38:16,569] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-08-08 18:38:16,570] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-08-08 18:38:16,685] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-08-08 18:38:16,812] - Script::install(): is successfully, Python=(3.4.2 (default, Aug  8 2018, 18:33:54) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-08-08 18:38:16,813] - Script::install(): KBExtra is successfully!
  INFO root [2018-08-08 18:38:16,865] - Dbmgr::initializeEnd: digest(363BB7D9C7C9010E5779213C13D38542)
S_INFO root [2018-08-08 18:38:16,866] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-08-08 18:38:16,867] - TelnetServer server is running on port 32000
  INFO root [2018-08-08 18:38:16,867] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2018-08-08 18:38:18,131] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
9CC9073E4B3526C053A1272B7D43CDB130CD5143AB5325957CD6428FE542E0BA773A1BF36028C3B9DA1E2A1CCBCEA85C96D7CE3FF9387B2B0B2FF2632262536A9027183BC73C2CFE3C2B090CBF2F3FAA2CEEFF79D89FC7A698CEA4F2485E0A15A10AC36E06BA20F7731008744DAC9FA90F63EBD7A82412390F3798FA011CEA44
 DEBUG root [2018-08-08 18:38:18,134] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 18:38:18,161] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2018-08-08 18:38:18,162] - DBUtil::createInterface[0x17f138]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2018-08-08 18:38:18,162] - ThreadPool::createThreadPool: creating  threadpool...
 DEBUG root [2018-08-08 18:38:18,162] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 18:38:18,163] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 18:38:18,163] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 18:38:18,163] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
  INFO root [2018-08-08 18:38:18,163] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2018-08-08 18:38:18,164] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 18:38:18,168] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 18:38:18,168] - DBThread::onStart(): 0x64d970!
 DEBUG root [2018-08-08 18:38:18,169] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 18:38:18,169] - DBThread::onStart(): 0x64d8e8!
 DEBUG root [2018-08-08 18:38:18,169] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 18:38:18,169] - DBThread::onStart(): 0x64d860!
 DEBUG root [2018-08-08 18:38:18,170] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 18:38:18,170] - DBThread::onStart(): 0x64da80!
 DEBUG root [2018-08-08 18:38:18,170] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 18:38:18,171] - DBThread::onStart(): 0x64d9f8!
  INFO root [2018-08-08 18:38:18,269] - DBInterfaceMysql::query: successfully!
  WARN root [2018-08-08 18:38:18,269] - DBInterfaceMysql::checkErrors: not found Account table, reset kbe_* table...
  WARN root [2018-08-08 18:38:18,272] - DBInterfaceMysql::checkErrors: reset kbe_* table end!
  INFO root [2018-08-08 18:38:18,439] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 18:38:18,446] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 18:38:18,459] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 18:38:18,461] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 18:38:18,465] - DBInterfaceMysql::query: successfully!
 DEBUG root [2018-08-08 18:38:18,465] - syncToDB(): Account->sm_autoLoad(tinyint not null DEFAULT 0).
  INFO root [2018-08-08 18:38:18,466] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 18:38:18,466] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 18:38:18,466] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 18:38:18,501] - DBInterfaceMysql::query: successfully!
  WARN root [2018-08-08 18:38:18,502] - DBTask::process(): took 0.23 seconds
sql:(show index from tbl_Account)
  WARN root [2018-08-08 18:38:18,524] - d:\kbe\kbengine-2.2.0\kbe\src\server\dbmgr\dbmgr.cpp:307: Profile scriptCall took 1.66 seconds
  INFO root [2018-08-08 18:38:18,971] - ---- dbmgr is running ----
 DEBUG root [2018-08-08 18:38:18,971] - Components::process(): Request for the process of identity...
  INFO root [2018-08-08 18:38:26,186] - Components::findComponents: find logger(1)...
  INFO root [2018-08-08 18:38:26,268] - Components::findComponents: found logger, addr:192.168.0.108:62411
  INFO root [2018-08-08 18:38:26,269] - Components::addComponent: reset orderLog, uid=9028!
  INFO root [2018-08-08 18:38:26,269] - Components::addComponent[logger], uid=9028, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2018-08-08 18:38:26,270] - Forwarding logs to logger[192.168.0.108:62411/0/0/0]...
 ERROR root [2018-08-08 18:38:50,718] - Components::removeComponentByChannel: baseappmgr : 5000, Abnormal exit! errorno=10054
  INFO root [2018-08-08 19:46:54,937] - 
  INFO root [2018-08-08 19:46:54,938] - -----------------------------------------------------------------------------------------


  INFO root [2018-08-08 19:46:54,940] - Resmgr::initialize: KBE_ROOT=D:/KBE/kbengine-2.2.0/
  INFO root [2018-08-08 19:46:54,940] - Resmgr::initialize: KBE_RES_PATH=D:/KBE/kbengine-2.2.0/kbe/res/;D:/KBE/kbengine-2.2.0/assets/;D:/KBE/kbengine-2.2.0/assets/scripts/;D:/KBE/kbengine-2.2.0/assets/res/
  INFO root [2018-08-08 19:46:54,940] - Resmgr::initialize: KBE_BIN_PATH=D:/KBE/kbengine-2.2.0/kbe/bin/server/
  INFO root [2018-08-08 19:46:54,963] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.108 ( 192.168.0.108:63658 )
  INFO root [2018-08-08 19:46:54,963] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.108:63658, SOMAXCONN=5.
  INFO root [2018-08-08 19:46:54,963] - server-configs:
  INFO root [2018-08-08 19:46:54,963] - 	internalTcpAddr : 192.168.0.108:63658
  INFO root [2018-08-08 19:46:54,963] - 	componentID : 4000
  INFO root [2018-08-08 19:46:56,470] - ---- dbmgr Version: 2.2.0. ScriptVersion: 0.1.0. Protocol: C85D8893D1079B3F8EA4D6F8485316E8. Config: Debug X86. Built: 18:34:10 Aug  8 2018. AppID: 4000. UID: 22485. PID: 65736 ----
  INFO root [2018-08-08 19:46:56,471] - Server : Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz with 4194303kB RAM
  INFO root [2018-08-08 19:46:56,471] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-08-08 19:46:56,472] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-08-08 19:46:56,623] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-08-08 19:46:56,623] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-08-08 19:46:56,624] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-08-08 19:46:56,629] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-08-08 19:46:56,629] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-08-08 19:46:56,630] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-08-08 19:46:56,631] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-08-08 19:46:56,691] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-08-08 19:46:56,753] - Script::install(): is successfully, Python=(3.4.2 (default, Aug  8 2018, 18:33:54) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-08-08 19:46:56,753] - Script::install(): KBExtra is successfully!
  INFO root [2018-08-08 19:46:56,764] - Dbmgr::initializeEnd: digest(363BB7D9C7C9010E5779213C13D38542)
S_INFO root [2018-08-08 19:46:56,764] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-08-08 19:46:56,764] - TelnetServer server is running on port 32000
  INFO root [2018-08-08 19:46:56,765] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2018-08-08 19:46:57,527] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
5EB1DDE3124E335920C092EF2372534981B62D5E0CD6DDF0BDC50340E5E0F1276AA1ED4254093483F418123142102CF560FE14EDD0E9B0A16A3C809168FFC22399F34438591EEE748DF9ECBEE76BE2DAFB28E9F580DE8BD64A7384E3BB8657FE63D1CDEBCBFFBC243CED0928E39CC45F39310101C168069DFB2823A968C42B2D
 DEBUG root [2018-08-08 19:46:57,529] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 19:46:57,534] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2018-08-08 19:46:57,534] - DBUtil::createInterface[0x188f348]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2018-08-08 19:46:57,534] - ThreadPool::createThreadPool: creating  threadpool...
 DEBUG root [2018-08-08 19:46:57,535] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 19:46:57,535] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 19:46:57,535] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 19:46:57,535] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
  INFO root [2018-08-08 19:46:57,535] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2018-08-08 19:46:57,535] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-08-08 19:46:57,539] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 19:46:57,539] - DBThread::onStart(): 0x1a3dad8!
 DEBUG root [2018-08-08 19:46:57,539] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 19:46:57,540] - DBThread::onStart(): 0x1a3db60!
 DEBUG root [2018-08-08 19:46:57,540] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 19:46:57,540] - DBThread::onStart(): 0x1a3dbe8!
 DEBUG root [2018-08-08 19:46:57,540] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 19:46:57,541] - DBThread::onStart(): 0x1a3dc70!
 DEBUG root [2018-08-08 19:46:57,541] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-08-08 19:46:57,541] - DBThread::onStart(): 0x1a3da50!
  INFO root [2018-08-08 19:46:57,636] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 19:46:57,637] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 19:46:57,637] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 19:46:57,637] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 19:46:57,637] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 19:46:57,637] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 19:46:57,638] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 19:46:57,638] - DBInterfaceMysql::query: successfully!
  INFO root [2018-08-08 19:46:57,638] - DBInterfaceMysql::query: successfully!
  WARN root [2018-08-08 19:46:57,657] - d:\kbe\kbengine-2.2.0\kbe\src\server\dbmgr\dbmgr.cpp:307: Profile scriptCall took 0.89 seconds
  INFO root [2018-08-08 19:46:57,999] - ---- dbmgr is running ----
 DEBUG root [2018-08-08 19:46:57,999] - Components::process(): Request for the process of identity...
  INFO root [2018-08-08 19:47:05,185] - Components::findComponents: find logger(1)...
  INFO root [2018-08-08 19:47:05,249] - Components::findComponents: found logger, addr:192.168.0.108:63653
  INFO root [2018-08-08 19:47:05,249] - Components::addComponent: reset orderLog, uid=22485!
  INFO root [2018-08-08 19:47:05,249] - Components::addComponent[logger], uid=22485, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2018-08-08 19:47:05,250] - Forwarding logs to logger[192.168.0.108:63653/0/0/0]...
 ERROR root [2018-08-08 19:49:35,981] - Components::removeComponentByChannel: loginapp : 9000, Abnormal exit! errorno=10054
 ERROR root [2018-08-08 19:49:38,177] - Components::removeComponentByChannel: baseapp : 7001, Abnormal exit! errorno=10054
 ERROR root [2018-08-08 19:49:40,075] - Components::removeComponentByChannel: cellapp : 8001, Abnormal exit! errorno=10054
 ERROR root [2018-08-08 19:49:46,285] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-08-08 19:49:46,289] - The following logs sent to logger failed:
  WARN root [2018-08-08 19:49:46,289] - ==> [2018-08-08 19:49:46 284] TCPPacketReceiver::processPendingEvents(192.168.0.108:63653): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-08-08 19:49:46,289] - ==> [2018-08-08 19:49:46 285] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
